src/gg.regression.unity.bots/Runtime/Scripts/StateRecorder/BotSegments/AIServiceManager.cs

```diff
@@ -175,7 +175,14 @@ private Task SendWebRequest(
                 },
                 contentType);
 
-        private async Task SendWebRequest(string uri, string method, string authToken, string payload, Func<Action, Task> abortRegistrationHook, Func<string, Task> onSuccess, Func<string, Task> onFailure, string contentType = "application/json")
+        private async Task SendWebRequest(string uri,
```

Comment:
whoops. Accidentally committed this local change I made for better readability on narrower screens.

Reply:
seems fine